packages/anima-cli/e2e-test/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { execSync } from 'child_process';
+
+const token = process.env.TEST_TOKEN; // personal team created for it
+if (!token) throw new Error('Anima team token for test not found');
+const storyHashReg = new RegExp(/hash => (.*)/);
+
+const tokenVersionMap = {
```

Comment:
it would be best to move it to another file 
```suggestion
const DESIGN_TOKENS_TEST_FILES = {
INIT: './e2e-test/test-ds-tokens.json',
UPDATE: './e2e-test/test-ds-tokens-update.json'
}
```